plugin-cgroupv2/src/k8s/token.rs

```diff
@@ -245,12 +245,7 @@ mod tests {
     // Test `refresh` function with valid file and token
     #[tokio::test]
     async fn test_refresh_with_valid_token() {
-        let tmp = tempdir().unwrap();
-        let root = tmp.path().join("test-alumet-plugin-k8s/var_1/");
-
-        if root.exists() {
-            std::fs::remove_dir_all(&root).unwrap();
-        }
+        let root = tempdir().unwrap().into_path();
```

Comment:
Beware, `into_path` consumes the TempDir and disables auto-cleaning. It has been deprecated and renamed to `keep` in recent versions of `tempfile`. If we want to clean the directories after the test (which I think is a good idea), we should keep the tempdir on its own and get its path with `path`, in another variable.